pkg/cmd/server/origin/asset.go

```diff
@@ -22,6 +22,10 @@ import (
 // then returns an array of strings indicating what endpoints were started
 // (these are format strings that will expect to be sent a single string value).
 func (c *AssetConfig) InstallAPI(container *restful.Container) []string {
+	if c.WebConsoleDisabled {
```

Comment:
doesn't belong here... just skip building the AssetConfig entirely if web console is disabled